Add subprojects documentation (#118)
karmada-io/community: GOVERNANCE.md

released synchronously with the main version when necessary. Among other listed projects, some are for exploratory 
purposes while others are related to peripheral ecosystem products.

Current Sub-Projects:
- [Dashboard](https://github.com/karmada-io/dashboard): Karmada Dashboard is a general-purpose, web-based control 
panel for Karmada.

The Karmada organization is open to receive new subprojects under its umbrella. To accept a project
into the __Karmada__ organization, it has to meet the following criteria:

- Must be licensed under the terms of the Apache License v2.0.
- Must be closely related to one or more areas of the Karmada ecosystem, and Karmada relies heavily on these projects.
- Must be supported by a Maintainer not associated or affiliated with the author(s) of the subprojects.
- Sub-Projects can have their own repositories but follow the same governance mechanism as the main project.
- Joining a subproject requires submitting an issue in the main project to obtain a 2/3 vote of approval from the Maintainers. Similarly, significant actions such as project archiving also require the consent of the Maintainers.

> The submission process starts as a Pull Request or Issue on the

Reviewer: ```suggestion
The submission process starts as an Issue on the
```
